src/app/app.module.ts

```diff
@@ -18,6 +18,9 @@ import { EnterResetPasswordComponent } from './pages/enter-reset-password/enter-
 import { AuthGuard } from './guards/auth.guard';
 import { NoAuthGuard } from './guards/no-auth.guard';
 
+
```

Comment:
@patrickf949 It does not look good to have blank lines in between import statements